docs/GetStarted/setup.md

```diff
@@ -123,25 +123,23 @@ Set the shared secret used in Kong for JWT authentication.
     !!! success
         ``` bash
         IMAGE                               STATUS                        NAMES
-        intel/oact-rps:v2.2.0               Up 2 minutes (healthy)        open-amt-cloud-toolkit_rps_1      
+        intel/oact-rps:latest               Up 2 minutes (healthy)        open-amt-cloud-toolkit_rps_1      
         eclipse-mosquitto                   Up 2 minutes                  open-amt-cloud-toolkit_mosquitto_1
         vault                               Up 2 minutes                  open-amt-cloud-toolkit_vault_1    
-        intel/oact-mpsrouter:v2.0.0         Up 2 minutes (healthy)        open-amt-cloud-toolkit_mpsrouter_1
-        sslpostgres                         Up 2 minutes (healthy)        open-amt-cloud-toolkit_db_1       
-        intel/oact-webui:v2.2.0             Up 2 minutes                  open-amt-cloud-toolkit_webui_1    
+        intel/oact-mpsrouter:latest         Up 2 minutes (healthy)        open-amt-cloud-toolkit_mpsrouter_1
+        postgres:14                         Up 2 minutes (healthy)        open-amt-cloud-toolkit_db_1       
+        intel/oact-webui:latest             Up 2 minutes                  open-amt-cloud-toolkit_webui_1    
         kong:2.3                            Up 2 minutes (healthy)        open-amt-cloud-toolkit_kong_1     
-        intel/oact-mps:v2.2.0               Up 2 minutes (healthy)        open-amt-cloud-toolkit_mps_1
+        intel/oact-mps:latest               Up 2 minutes (healthy)        open-amt-cloud-toolkit_mps_1
         ```
-
   
-!!! important 
-    ** Container Issues **
+        !!! warning "Warning - Container Issues" 
```

Comment:
Warning shouldn't be inside success

![image](https://user-images.githubusercontent.com/22204562/194166846-943530b8-6d01-4384-9b79-6c924c1bb45a.png)